### PR TITLE
fix: 🐛 Drop foreign key constraint for `Leg.assetId`

### PR DESCRIPTION
**Pull request by prashantasdeveloper**
### Description

As of now, chain doesn't validate the Legs to have a valid Asset ticker. This will be handled in on of the chain's future releases. To solve this now, we are dropping the foreign key constraint on the asset attribute of Leg entity. 

This problem was found in block 6101310 on testnet, erring as follows - 
```
2022-12-13T12:28:25.053Z <sandbox> ERROR Received an error in handleEvent while handling the event 'settlement.InstructionCreated': Error
    at Query.run (/usr/local/lib/node_modules/@subql/node/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)
    at /usr/local/lib/node_modules/@subql/node/node_modules/sequelize/lib/sequelize.js:313:28
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async PostgresQueryInterface.upsert (/usr/local/lib/node_modules/@subql/node/node_modules/sequelize/lib/dialects/abstract/query-interface.js:330:12)
    at async Function.upsert (/usr/local/lib/node_modules/@subql/node/node_modules/sequelize/lib/model.js:1501:20)
    at async Object.set (/usr/local/lib/node_modules/@subql/node/dist/indexer/store.service.js:299:17) 
2022-12-13T12:28:25.055Z <fetch> ERROR failed to index block at height 6101310 handleEvent() SequelizeForeignKeyConstraintError: insert or update on table "legs" violates foreign key constraint "legs_asset_id_fkey"
```

### Breaking Changes

NA

### JIRA Link

NA

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -1361,7 +1361,7 @@ enum InstructionStatusEnum {
 
 type Leg @entity {
   id: ID! # instructionId/legIndex
-  asset: Asset!
+  assetId: String! # @prashantasdeveloper Replace this with `asset: Asset` when chain validations are fixed
```
Comment (polymath-eric) [inline schema.graphql:1364]: Could this be `Asset`, as in nullable reference?

Even when chain validations are in place we still need to support the potential references made before then.

File: db/migrations/8.5.04.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+alter table legs drop constraint if exists legs_asset_id_fkey;
```
Comment (polymath-eric) [inline db/migrations/8.5.04.sql:1]: file should be `8.5.4`

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "polymesh-subquery",
-  "version": "8.5.0",
+  "version": "8.5.04",
```
Comment (polymath-eric) [inline package.json:3]:
```suggestion
  "version": "8.5.4",
```
why `04`?

File: schema.graphql (hunk 1 above)
Comment (prashantasdeveloper) [reply schema.graphql:1364]: We can change this to support null when we have that in place. For now lets stick to it

**Review by polymath-eric — APPROVED** [review]

**Pull request merged**